benchmarks/README.md

```diff
@@ -12,17 +12,20 @@ repeated 10 times.
 ------------------|------------|-----------|---------------------|--------------------|-----------------------
 |Branin| 0.398 +/- 0.000 |0.398 | 33.1 | 5.7 | 27
 |Hart6| -3.299 +/- 0.048|-3.322 | 57.3 | 5.658 | 47
+|ML_hypp_tune| -0.860 +/- 0.045 | -0.898 | NA | NA | NA
```

Comment:
No, it is just that the GP makes a prior assumption that `f(x)` follows a multivariate gaussian and hence a continuous function is easier to model for the GP. However I don't think that is a concern for tree-based methods.

It would be interesting to try both and see if tree-based methods have a significant advantage if the function is discontinuous.